tests/container/classloading_in_deconstruct/src/main/java/com/yahoo/vespatest/DeconstructSearcher.java

```diff
@@ -35,7 +35,8 @@ public void deconstruct() {
         try {
             Class<?> causesException = this.getClass().forName("com.yahoo.docproc.DocumentProcessor");
         } catch (ClassNotFoundException e) {
-            log.log(Level.SEVERE, "Class not found when deconstructing searcher", e);
+            log.log(Level.SEVERE, "Class not found when deconstructing searcher " + this
```

Comment:
I thought the keyword was `self, not `this`

Reply:
This is java :)